src/OrbitGl/App.cpp

```diff
@@ -1413,6 +1415,34 @@ static std::unique_ptr<CaptureEventProcessor> CreateCaptureEventProcessor(
   return CaptureEventProcessor::CreateCompositeProcessor(std::move(event_processors));
 }
 
+static void FindAndAddFunctionToStopUnwindingAt(
+    const std::basic_string<char>& function_name, const std::basic_string<char>& module_name,
+    const orbit_client_data::ModuleManager& module_manager, const ProcessData& process,
+    std::map<uint64_t, uint64_t>* absolute_address_to_size_of_functions_to_stop_unwinding_at) {
+  std::vector<orbit_client_data::ModuleInMemory> modules =
+      process.FindModulesByFilename(module_name);
+  for (const auto& module_in_memory : modules) {
+    const ModuleData* module_data = module_manager.GetModuleByPathAndBuildId(
+        module_in_memory.file_path(), module_in_memory.build_id());
+
+    const FunctionInfo* function_to_stop_unwinding_at =
+        module_data->FindFunctionFromPrettyName(function_name);
+    if (function_to_stop_unwinding_at == nullptr) {
+      continue;
+    }
+    uint64_t function_absolute_start_address =
+        orbit_object_utils::SymbolVirtualAddressToAbsoluteAddress(
+            function_to_stop_unwinding_at->address(), module_in_memory.start(),
+            module_data->load_bias(), module_data->executable_segment_offset());
+
+    auto [unused_it, inserted] =
+        absolute_address_to_size_of_functions_to_stop_unwinding_at->insert_or_assign(
+            function_absolute_start_address, function_to_stop_unwinding_at->size());
+    ORBIT_CHECK(inserted);
+    break;
```

Comment:
Why `break`? Didn't we say that if the module is loaded multiple times we want to send address+size just as many times?

Reply:
Good point. Removed it.